src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -560,17 +566,67 @@ function tryRemoveElement(elem) {
             return relativeOffset;
         }
 
+        /**
+         * @private
+         * These browsers will not clear the existing active cue when setting an offset
+         * for native TextTracks.
+         * Any previous text tracks that are on the screen when the offset changes will remain next
+         * to the new tracks until they reach the end time of the new offset's instance of the track.
+         */
+        requiresHidingActiveCuesOnOffsetChange() {
+            return !!browser.firefox;
+        }
+
+        /**
+         * @private
+         */
+        hideTextTrackWithActiveCues(currentTrack) {
+            if (currentTrack.activeCues) {
+                currentTrack.mode = 'hidden';
+            }
+        }
+
+        /**
+         * Forces the active cue to clear by disabling then re-enabling the track.
+         * The track mode is reverted inside of a 0ms timeout to free up the track
+         * and allow it to disable and clear the active cue.
+         * The track needs to be temporarily stored in order for us to access it
+         * inside the timeout. The stored value is reset after it is used.
+         * @private
+         */
+        forceClearTextTrackActiveCues(currentTrack) {
+            if (currentTrack.activeCues) {
+                this.#currentTextTrack = currentTrack;
```

Comment:
I originally stored it temporarily because I was oddly getting strange errors where it was losing the reference. But I'm trying this again by just referencing the `currentTrack` that we pass in, and I am not getting those errors anymore.
You are correct, the closure is keeping access open for us.
The error I was getting in the console before must have been something unrelated.

Reply:
Thanks for questioning this and getting me to look at it again.

Reply:
I confirmed that this PR fix in it's current state works with the secondary subtitles.